<launch>

  <!-- params -->
  <param name="forest_file" value="$(find leg_tracker)/config/trained_leg_detector_res=0.33.yaml" />
  <param name="scan_topic" value="/scan" />
  <param name="fixed_frame" value="odom"/>
  <param name="scan_frequency" value="15"/>  
  <param name="max_detect_distance" value="7.5"/>
  
  <!-- run detect_leg_clusters -->  
  <node pkg="leg_tracker" type="detect_leg_clusters" name="detect_leg_clusters" output="screen"/>

  <!-- run joint_leg_tracker -->
  <node pkg="leg_tracker" type="joint_leg_tracker.py" name="joint_leg_tracker" output="screen"/>
 
   <!-- run local_occupancy_grid_mapping -->
  <node pkg="leg_tracker" type="local_occupancy_grid_mapping" name="local_occupancy_grid_mapping" output="screen"/> 
   
</launch>

